### DISCUSSION TITLE
What does the red dot above the tag cloud icon mean?

### DISCUSSION THREAD

**Discussion by rpace314159** [Zettlr/Zettlr#2860] [Q&A] [answered]
Hi: I see a red dot above the tag cloud icon sometimes.  What does that mean?  
![image](https://user-images.githubusercontent.com/93452310/143622427-81407c99-181f-49d7-b4d9-37cda91fa6e1.png)

**Comment by nathanlesage** [comment, marked as answer]
This indicates that there are words contained in the currently active file which are also tags, so the app is suggesting tags. If there are no suggested tags for the current file, the dot disappears. I haven't found a smarter way to do that unfortunately.
